libcaf_core/caf/flow/op/on_error_resume_next.test.cpp

```diff
@@ -14,7 +14,6 @@ using namespace caf;
 namespace {
 
 WITH_FIXTURE(test::fixture::flow) {
-
 SCENARIO("on_error_resume_next switches to a fallback observable on error") {
```

Comment:
Did you remove that on purpose? We separate the fixture scope from the tests/scenarios with a vertical space.